index.html

```diff
@@ -2,14 +2,76 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 </head>
 <body>
+<nav class="nav-bar">
+    <div class="links">
+        <img class="logo" src="img/karma-logo.svg" width="26">
+        <ul class="top">
+            <li class="active"><a href="#">Meet Karma</a></li>
+            <li>How it Works</li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+            </ul>
+    </div>
+</nav>
+<section class="main">
+    <div class="introduction">
+        <h1>Introducting Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <div class="button">
+            <button>Learn More</button>
+        </div>
+</section>
+</div>
+<section>
+    <div class="container">
+        <h2>Everyone needs a little Karma</h2>
+        <div class="row justify-content-between align-items-center">
+            <div class="col-lg-4 col-md-3 col-sm-1 icons">
+                <img src="img/icon-devices.svg" alt="Internet for all divices image" width=127 height=127>
+                <h3>Internet for all divices</h3>
+            </div>
+            <div class="col-lg-4 col-md-3 col-sm-1 icons">
+                <img src="img/icon-coffee.svg" alt="Boots your productivity image" width=127 height=127>
+                <h3>Boost your productivity</h3>
+            </div>
+            <div class="col-lg-4 col-md-3 col-sm-1 icons">
+                <img src="img/icon-refill.svg" alt="Pay as You Go image" width=127 height=127>
+                <h3>Pay as You Go</h3>
+            </div>
+        </div>
+    </div>
+</section>
+<footer class="social">
+    <div class="content">
+        <hr>
+        <h5>Join us on</h5>
+        <ul>
```

Comment:
remember `<ul>`s have a default `padding-left`. Set it to 0 and your icons will be centered.